tools/cli/commands.js

```diff
@@ -325,6 +325,15 @@ function doRunCommand(options) {
     appPort = appPortMatch[2] ? parseInt(appPortMatch[2]) : null;
   }
 
+  if (options.production) {
+    Console.warn(
```

Comment:
This is good but the messaging could do with some grammatical tweaks. I'd suggest 

"Warning: The --production flag should only be used to simulate production bundling for testing purposes. Use `meteor build` to create a bundle for production deployment (see: https://guide.meteor.com/deployment.html)".